p2p/peer_test.go

```diff
@@ -17,9 +17,12 @@
 package p2p
 
 import (
+	"bytes"
 	"encoding/binary"
 	"errors"
 	"fmt"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/stretchr/testify/require"
```

Comment:
these should be added to the import-paragraph further down, not up here

Reply:
These changes are not present anymore